### PR TITLE
Merge remote-tracking branch 'origin/releases' into mc/merge/1.0_42b1a269

### PR DESCRIPTION
**Pull request by beeender**

### CODE REVIEW

File: realm/realm-library/src/main/java/io/realm/internal/TableView.java [hunk 1]
```diff
@@ -724,17 +724,25 @@ public String toJson() {
 
     @Override
     public String toString() {
-        return nativeToString(nativePtr, 500);
-    }
+        long columnCount = getColumnCount();
+        StringBuilder stringBuilder = new StringBuilder("The TableView ");
+        stringBuilder.append("contains ");
```
Comment (spl) [inline realm/realm-library/src/main/java/io/realm/internal/TableView.java]: I suppose `"contains "` could be combined with `"The TableView "` in the `StringBuilder` constructor, but it's a minor thing.

Comment (beeender) [inline realm/realm-library/src/main/java/io/realm/internal/TableView.java]: Thanks! will fix it!

**Comment by cmelchior** [comment]
👍

**Comment by stk1m1** [comment]
👍

**Pull request merged**